distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -166,18 +164,20 @@
     @BeforeExam
     public void beforeExam() throws Exception {
         try {
+            console = new KarafConsole(getServiceManager().getBundleContext(),
```

Comment:
❓ Why not move this to `AbstractIntegrationTest`? Or create a method like
```java
protected void runConsoleCommand(String command);
```
in `AbstractIntegrationTest`?